pibooth/plugins/lights_plugin.py

```diff
@@ -2,83 +2,100 @@
 
 import time
 import pibooth
+from pibooth.utils import LOGGER
+from gpiozero import Device, LEDBoard
+try:
+    from RPi import GPIO
+    LOGGER.info("Start on Raspberry pi")
+except ImportError:
+    # GPIO library only existes on Raspberry Pi
+    # Set the default pin factory to a mock factory
+    from gpiozero.pins.mock import MockFactory
+    Device.pin_factory = MockFactory()
+    LOGGER.info("Programme not start on Raspberry pi use MockFactory() mode")
 
 
 class LightsPlugin(object):
 
     """Plugin to manage the lights via GPIO.
     """
+    def __init__(self):
+        self.blink_time = 0.3
+        self.load = False
+
+    def start(self, cfg):
+
+        if not self.load:
+            self.leds = LEDBoard(
+                            led_capture="BOARD" + cfg.get('CONTROLS', 'picture_led_pin'),
+                            led_print="BOARD" + cfg.get('CONTROLS', 'print_led_pin'),
+                            led_preview="BOARD" + cfg.get('CONTROLS', 'preview_led_pin'),
+                            led_start="BOARD" + cfg.get('CONTROLS', 'startup_led_pin'),
+                            # pwm=True
+                    )
+            self.leds.led_start.on()
+            self.load = True
 
     @pibooth.hookimpl
-    def pibooth_startup(self, app):
-        app.led_startup.switch_on()
-
-    @pibooth.hookimpl
-    def pibooth_cleanup(self, app):
-        app.led_startup.quit()
-        app.led_preview.quit()
-        app.led_capture.quit()
-        app.led_print.quit()
-
-    @pibooth.hookimpl
-    def state_wait_enter(self, app):
-        app.led_capture.blink()
+    def state_wait_enter(self, cfg, app):
+        self.start(cfg)
+        self.leds.led_capture.blink(on_time=self.blink_time, off_time=self.blink_time, n=None, background=True)
```

Comment:
The parameters `n=None, background=True` are the default values, thus this is useless to define them.